yawl/lib/yawl/widget/Histogram.lua

```diff
@@ -0,0 +1,146 @@
+local class = require("libClass2")
+local Frame = require("yawl.widget.Frame")
+local gpu = require("component").gpu
+
+--[[
+    Ideally, Histogram will have horizontal scrolling. Though there is some contention about the direction.
+]]
+---@class Histogram:Frame
+---@field parent Frame
+---@operator call:Histogram
+---@overload fun(parent:Frame,x:number,y:number)
+local Histogram = class(Frame)
```

Comment:
Are you sure this is a `Frame` and not a `Widget` ?
It doesn't look like it is meant to hold widgets

Reply:
Not sure yet, made it into a frame first in the event extra widgets may be added